_board/microdev_micro_c3.md

```diff
@@ -1,17 +1,40 @@
 ---
 layout: download
 board_id: "microdev_micro_c3"
-title: "microC3 Download"
-name: "microC3"
-manufacturer: "microDev"
-board_url: ""
-board_image: "unknown.jpg"
-date_added: 2021-11-15
+title: "MicroDev microC3 Download"
+name: "MicroDev microC3"
+manufacturer: "MicroDev"
+board_url: "https://microdev.systems/"
+board_image: "microdev_micro_c3.jpg"
+date_added: 2021-10-06
 family: esp32c3
 bootloader_id: microdev_micro_c3
 downloads_display: false
```

Comment:
You'll also want to remove the `downloads_display: false` or change it to true

Reply:
Done! :-)